app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.java

```diff
@@ -181,12 +181,20 @@ public static void init(Context context) {
         }
     }
 
-    private static SettingsMeasurement.SharedPreferenceSettingsProvider makeSettingsProvider(final Resources resources) {
+    private static SettingsMeasurement.SharedPreferenceSettingsProvider makeSettingsProvider(final Context context, final Resources resources) {
```

Comment:
\[PMD\] INFO: The method 'makeSettingsProvider' has a Cyclomatic Complexity of 8\.